fundraiser/project/fundraiser-contract/tests/functions/core/pledge.rs

```diff
@@ -377,11 +463,11 @@ mod revert {
     }
 
     #[tokio::test]
-    #[ignore]
     #[should_panic(expected = "CampaignEnded")]
     async fn when_pledging_after_deadline() {
         let (author, user, asset, _, defaults) = setup().await;
-        let deadline = 5;
+        let provider = author.wallet.get_provider().unwrap();
+        let deadline = provider.latest_block_height().await.unwrap() + 3;
```

Comment:
see my other comment, but I would consider using the `produce_blocks` function for this test case